client/client/app/shared/projectContext/index.js

```diff
@@ -2148,6 +2184,14 @@ export default class projectContext {
         }
     }
 
+    getVcfSampleNames () {
+        const names = this._vcfFilter ? this._vcfFilter.sampleNames : [];
+        const sampleNames = names.map(name => (
+            [this._vcfSampleAliases[name]].join(', ') || name
```

Comment:
```js
[this._vcfSampleAliases[name]].join(', ')
```
extra array is redundant - we want to join all elements within `this._vcfSampleAliases[name]`, but here we join *an array of array*